bin/download

```diff
@@ -10,10 +10,10 @@ source "${plugin_dir}/lib/utils.bash"
 
 mkdir -p "$ASDF_DOWNLOAD_PATH"
 
-release_file="$ASDF_DOWNLOAD_PATH/$RELEASE_NAME-v$ASDF_INSTALL_VERSION.tar.gz"
+release_file="$ASDF_DOWNLOAD_PATH/$RELEASE_NAME-$ASDF_INSTALL_VERSION.tar.gz"
```

Comment:
I think removing the `v` breaks compatibility with how projects currently set the version number.

e.g. https://github.com/AdaptiveConsulting/platform-exchange/blob/develop/.envrc#L28

Reply:
I think the project gets it wrong in that case. The question is if we want to force the change everywhere. Probably not, so maybe let's keep the v for now as @bersalazar mentions.

Reply:
this is the "download" file - removing the v and filtering the _VERSION_ parameter is needed (because it might be a `ref=` which may include invalid filesystem characters (e.g. `/` `:` etc.)

Reply:
the `v` then needs to come from the consumer side, which means changing .envrc in every consumer.
if we want to support `ref=` type then we should probably form the URL based on ASDF_INSTALL_TYPE.

Reply:
I'll be removing the v prefix if ever it is specified during ASDF_INSTALL_TYPE=VERSION

	`if [ "$install_type" == "version" ]; then
		url="${GITHUB_REPO_URL}/archive/refs/tags/v${version#v}.tar.gz"
	else
		url="${GITHUB_API_URL}/repos/${OWNER}/${REPO}/tarball/${version}"
	 fi`